tracetools/propagate.go

```diff
@@ -15,15 +15,16 @@ const EnvVarTraceContextKey = "BUILDKITE_TRACE_CONTEXT"
 
 // EncodeTraceContext will serialize and encode tracing data into a string and place
 // it into the given env vars map.
```

Comment:
I'd really prefer to not be changing this file at all. Unfortunately the otel provided OpenTracing shim doesn't support the `opentracing.TextMap` format